sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -201,6 +223,27 @@ public AutoConfiguredOpenTelemetrySdk build() {
         LogEmitterProviderConfiguration.configureLogEmitterProvider(
             resource, config, meterProvider);
 
+    if (registerShutdownHook) {
+      Runtime.getRuntime()
+          .addShutdownHook(
+              new Thread(
+                  () -> {
+                    List<CompletableResultCode> shutdown = new ArrayList<>();
+                    shutdown.add(tracerProvider.shutdown());
+                    if (meterProvider instanceof SdkMeterProvider) {
+                      shutdown.add(((SdkMeterProvider) meterProvider).shutdown());
+                    }
+                    shutdown.add(logEmitterProvider.shutdown());
+                    CompletableResultCode.ofAll(shutdown).join(10, TimeUnit.SECONDS);
+                  }));
+
+      Runtime.getRuntime().addShutdownHook(new Thread(tracerProvider::close));
```

Comment:
Looks like forgot to delete the old code with three threads

Reply:
Ops. Sorry. Fixed now.